src/features/world/ui/flowerShop/FlowerBounties.tsx

```diff
@@ -0,0 +1,254 @@
+                      style={{
+                        right: `${PIXEL_SCALE * -3}px`,
+                        // width: `calc(100% + ${PIXEL_SCALE * 6}px)`,
+                        height: "25px",
+                      }}
+                    >
+                      {t("bounties.sold")}
+                    </Label>
+                  )}
+
+                  {!!deal.coins && (
+                    <Label
+                      type="warning"
+                      icon={SUNNYSIDE.ui.coinsImg}
+                      className={"absolute -bottom-2 text-center p-1 "}
+                      style={{
+                        left: `${PIXEL_SCALE * -3}px`,
+                        right: `${PIXEL_SCALE * -3}px`,
+                        width: `calc(100% + ${PIXEL_SCALE * 6}px)`,
+                        height: "25px",
+                      }}
+                    >
+                      {deal.coins}
+                    </Label>
+                  )}
+
+                  {getKeys(deal.items ?? {}).map((name) => {
+                    return (
+                      <Label
+                        key={name}
+                        type="warning"
+                        icon={ITEM_DETAILS[name].image}
+                        className={"absolute -bottom-2 text-center p-1 "}
+                        style={{
+                          left: `${PIXEL_SCALE * -3}px`,
+                          right: `${PIXEL_SCALE * -3}px`,
+                          width: `calc(100% + ${PIXEL_SCALE * 6}px)`,
+                          height: "25px",
+                        }}
+                      >
+                        {deal.items?.[name]}
+                      </Label>
+                    );
+                  })}
+                </ButtonPanel>
+              </div>
+            );
+          })}
+        </div>
+      </div>
+    </CloseButtonPanel>
+  );
+};
+
+const Deal: React.FC<{
+  deal: BountyRequest;
+  onClose: () => void;
+  onSold: () => void;
+}> = ({ deal, onClose, onSold }) => {
+  const { gameService } = useContext(Context);
+
+  const { t } = useAppTranslation();
+  const sell = () => {
+    gameService.send("bounty.sold", {
+      requestId: deal.id,
+    });
+
+    onSold();
+  };
+
+  if (!deal) {
+    return null;
+  }
+
+  return (
+    <Panel bumpkinParts={NPC_WEARABLES.poppy}>
+      <div className="p-2">
+        <div className="mb-2 flex flex-wrap">
+          <Label
+            type="default"
+            icon={ITEM_DETAILS[deal.name].image}
+            className="mr-2"
+          >
+            {deal.name}
+          </Label>
+          {!!deal.coins && (
+            <Label type="warning" icon={SUNNYSIDE.ui.coinsImg}>
+              {deal.coins}
+            </Label>
+          )}
+
+          {getKeys(deal.items ?? {}).map((name) => {
+            <Label key={name} type="warning" icon={ITEM_DETAILS[name].image}>
+              {deal.items?.[name]}
+            </Label>;
+          })}
```

Comment:
IDE picked this one up:
 
```suggestion
          {getKeys(deal.items ?? {}).map((name) => 
            <Label key={name} type="warning" icon={ITEM_DETAILS[name].image}>
              {deal.items?.[name]}
            </Label>
          )}
```